tests/test_alignment.py

```diff
@@ -169,10 +181,15 @@ def test_wcs_from_footprints():
 
     wcs = wcs_from_footprints([dm_1, dm_2])
 
-    # check that center of calculated WCS matches the
-    # expected position onto wcs_1 and wcs_2
-    assert all(np.isclose(wcs(2, 2), wcs_1(1, 1)))
-    assert all(np.isclose(wcs(2, 2), wcs_2(2, 2)))
+    # check that all elements of footprint match the *vertices* of the new combined WCS
+    assert all(np.isclose(wcs.footprint()[0], wcs(0, 0)))
```

Comment:
Just a suggestion: there is built in `np.testing.assert_allclose` that does the same thing as this function composition.

Reply:
Thanks for the tip, @kmacdonald-stsci! I'll adopt that method moving forward.